src/orchard/xref.clj

```diff
@@ -45,8 +81,40 @@
   "Find all functions that refer `var`.
   `var` can be a function value, a var or a symbol."
   {:added "0.5"}
-  [var]
-  (let [var (as-var var)
+  [v]
+  (let [var (as-var v)
         all-vars (q/vars {:ns-query {:project? true} :private? true})
         deps-map (zipmap all-vars (map fn-deps all-vars))]
     (map first (filter (fn [[_k v]] (contains? v var)) deps-map))))
+
+(comment
```

Comment:
Are those some functions you used for test purposes? Might be good idea to add a comment explaining this.